src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -25,10 +25,14 @@
 
 {% set paginationNum = 8 %}
 {% set isColumnLocationDisplayed = false %}
+{% set isColumnRefundedDisplayed = false %}
```

Comment:
I would be better if we could integrated this in the controller instead of the twig (same for `isColumnLocationDisplayed`)